frontend/src/components/Comments/CommentForm.vue

```diff
@@ -1,41 +1,68 @@
 <template>
-    <CardForm :title="`Submit ${type}`">
```

Comment:
Does CardForm get used anywhere else? Maybe that component should be reworked to the below.

Reply:
Workin' on it

Reply:
So I've ended up just using Card here... I think CardForm looks good for question & review still, so I don't want to change it.
I could have made CardFormSmall or something... but don't think it's worth it rn

Reply:
Oh yeah I forgot for NewView.